.circleci/config.yml

```diff
@@ -1,36 +1,56 @@
-# Javascript Node CircleCI 2.0 configuration file
-#
-# Check https://circleci.com/docs/2.0/language-javascript/ for more details
-#
 version: 2
 jobs:
-  build:
+  install:
+    working_directory: ~/repo
     docker:
-      # Specify service dependencies here if necessary
-      # CircleCI maintains a library of pre-built images
-      # documented at https://circleci.com/docs/2.0/circleci-images/
+      # https://circleci.com/docs/2.0/circleci-images/
       - image: node:12
-
-    working_directory: ~/repo
-
     steps:
       - checkout
-
-      # Download and cache dependencies
       - restore_cache:
-          keys:
-            - v1-dependencies-{{ checksum "package.json" }}
-            # fallback to using the latest cache if no exact match is found
-            - v1-dependencies-
-
-      - run: npm install
-
+          key: dependency-cache-{{ checksum "package-lock.json" }}
+      - run:
+          name: Install dependencies
+          command: npm ci
```

Comment:
`npm install` will look at versions like `^2.1.1` and see that there is a `2.1.2` available, download that, and update the `package-lock`. If there was a bug in `2.1.2` of whatever dependency that is, then your CI will fail while your local passes, which can be confusing. Instead of `npm install` it can run `npm ci` which looks at the `package-lock` and installs exactly the same versions as what you have locally.

Reply:
Brilliant! CI/CD is not setup for this project, actually. I just copied that file from a different project because I WANT to setup CI/CD eventually. Currently I'm still building on my own and publishing, but I DEFINITELY want to automate that. 

Not sure if CircleCI or GitHub Actions is the better approach, but this is still a nice addition.

Reply:
No clue, I've only ever used TravisCI and just to automate lint/unit testing compliance on PR's. I'd like to have everything else automated more, but never took the time to look into.

Reply:
Made an Issue for the CI/CD stuff since it is out of scope for this linting setup PR. I can revert this file or leave it as is.

Reply:
Yeah, I saw that. Thanks! You can leave this as is. I think I want to try out GH actions, so this file might just be deleted eventually. But Ive never set up actions, so it will be a learning curve.

On a different subject, I'm going to be preparing the next release soon and wanted to call out yoru support. What's the best way to do so? I found you on Twitter but doesnt seem very active. Should I just point to your website?

Reply:
Twitter is fine